testdata/sqllogictests/functions/hashing.slt

```diff
@@ -0,0 +1,108 @@
+statement error
+select siphash(1, 2, 3);
+
+statement error
+select siphash(1, 2);
+
+statement ok
+select siphash(1);
+
+statement ok
+select siphash('000');
+
+statement ok
+select siphash(9001);
+
+statement ok
+select siphash(true);
+
+
+statement error
+select fnv(1, 2, 3);
```

Comment:
I think this _technically_ violates the type signature of `TypeSignature::Any(1)`. Maybe we should use `TypeSignature::VariadicAny` instead